docs/source/datasets.rst

```diff
@@ -0,0 +1,72 @@
+Datasets
+========================
+To perform an automated systematic review, ASReview requires a dataset of papers.
+The dataset of papers usually results from an online library search for all studies related to the topic to review.
+All papers in the dataset need to be screened, with the goal to select all relevant papers. 
+
+For testing and demonstrating ASReview, the software offers `three built-in datasets <#built-in-demonstration-datasets>`__
+It is also possible to perform an automated systematic review on `your own dataset <#using-your-own-data>`__.
+Every paper in the dataset carries metadata such as title, abstract, author, et cetera. 
+
+Built-in demonstration datasets
+---------------------
+The built-in datasets come from 'traditional' systematic reviews on various research topics. 
+All papers who were selected in the original systematic review are indicated.
```

Comment:
who -> which

Reply:
what do you mean with 'traditional'? I would maybe refer to PRISMA based reviews?